src/StudyRegistration/StudyRegistrationRequestForm.tsx

```diff
@@ -294,6 +296,11 @@ const StudyRegistrationRequestForm: React.FunctionComponent<StudyRegistrationPro
               </Button>
             </Space>
           </Form.Item>
+          { (studyRegistrationConfig.studyRegistrationFormDisclaimerField) &&
+              <Typography style={{ textAlign: 'center' }}>
+                {studyRegistrationConfig.studyRegistrationFormDisclaimerField}
+              </Typography>
+          }
```

Comment:
```suggestion
          { (studyRegistrationConfig.studyRegistrationFormDisclaimerField)
              && (
                <Typography style={{ textAlign: 'center' }}>
                  {studyRegistrationConfig.studyRegistrationFormDisclaimerField}
                </Typography>
              )}
```

better to add parentheses around multilines JSX